tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +348,156 @@ def test_del_obj_tag_40436(self):
+                                   versions_dict=self.versions)
+        latest_v = self.versions[self.object_name]["version_history"][-1]
+        LOGGER.info(
+            "Step 2: Uploaded (creates version with versionId) of object - %s ",
+            self.object_name)
+        LOGGER.info("Step 3: Perform PUT Bucket versioning with status as Suspended")
+        res = self.s3_ver_obj.put_bucket_versioning(
+            bucket_name=self.bucket_name, status="Suspended")
+        assert_utils.assert_true(res[0], res[1])
+        LOGGER.info("Step 3: Performed PUT Bucket versioning with status as Suspended")
+        LOGGER.info("Step 4: Perform PUT Object for object1 (creates version with null versionId)")
+        s3_ver_tlib.upload_version(self.s3_test_obj,
+                                   bucket_name=self.bucket_name,
+                                   file_path=self.file_path2,
+                                   object_name=self.object_name,
+                                   versions_dict=self.versions,
+                                   chk_null_version=True)
+        LOGGER.info("Step 4: Performed PUT Object for object1(creates version with null versionId)")
+        ver_list = self.versions[self.object_name]["version_history"]  # [latest_v, "null"]
+        LOGGER.info("Step 5: Perform PUT Object Tagging for object1 with a tag key-value pair for "
+                    "versionIds %s and null version", latest_v)
+        self.ver_tag.update({self.object_name: dict()})
+        for v_id in ver_list:
+            resp = s3_ver_tlib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj, version_id=v_id,
+                                                  s3_ver_test_obj=self.s3_ver_obj,
+                                                  version_tag=self.ver_tag,
+                                                  bucket_name=self.bucket_name,
+                                                  object_name=self.object_name)
+            assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 5: Performed PUT Object Tagging for object1 with a tag key-value pair "
+                    "for versionIds %s and null version", latest_v)
+        LOGGER.info(
+            "Step 6: GET Object Tagging for %s with versionIds=%s",self.object_name, ver_list)
+        for v_id in ver_list:
+            resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                                  s3_ver_test_obj=self.s3_ver_obj,
+                                                  bucket_name=self.bucket_name,
+                                                  object_name=self.object_name,
+                                                  version_id=v_id)
+            assert_utils.assert_true(resp[0], resp)
+            get_tag = resp[1]
+            latest = self.versions[self.object_name]["version_history"][-1]
+            put_tag = self.ver_tag[self.object_name][latest]
```

Comment:
You will need to get tags for v_id here, not for latest

Reply:
Done